EU4toV2/Source/V2World/Output/outCountry.cpp

```diff
@@ -117,29 +117,52 @@ std::ostream& V2::operator<<(std::ostream& output, const Country& country)
 
 void V2::Country::outputCommons(std::ostream& output)
 {
-	output << "graphical_culture = UsGC\n";	// default to US graphics
+	output << "graphical_culture = ";
+	auto graphicalCulture = modCommons.getGraphicalCulture();
```

Comment:
You can probably rewrite this cleaner.
if (!modCommons.getGraphicalCulture().empty)
output << modCommons.getGraphicalCulture()<< '\n';
else if (!graphicalCulture.empty())
		output << graphicalCulture << '\n';
	else
		output << "UsGC\n"; // default to US graphics